### PR TITLE
CRYP-228: added ethereum transaction fee amount

### PR DESCRIPTION
**Pull request by Andre-Ibrahim**
image

### CODE REVIEW

File: packages/client/components/transaction-details/TransactionDetails.tsx [hunk 1]
```diff
@@ -258,7 +261,9 @@ export function TransactionDetails({ txn, wallet, navigation, otherDetails }: Pr
             />
             <MultiLineListItem
                 label={"Fee"}
-                value={`${getFormattedAmount(String(0), wallet.currencyType)} ${typeToISOCode[wallet.currencyType]}`}
+                value={`${getFormattedAmount(String(tranasctionFee), wallet.currencyType)} ${
```
Comment (jason-gerard) [inline packages/client/components/transaction-details/TransactionDetails.tsx:264]: I know its not your code but those type casting functions are no good

```suggestion
                value={`${getFormattedAmount(tranasctionFee.toString(), wallet.currencyType)} ${
```

**Review by jason-gerard — APPROVED** [review]
Why all the .toLowerCase()? does our APIs not respect casing?

**Comment by Andre-Ibrahim** [comment]
> Why all the .toLowerCase()? does our APIs not respect casing?

I believe alchemy sends all the addresses in lowercase so that list was empty since the filter was never set to true

**Review by narroarrow — APPROVED** [review]

**Review by domsec — APPROVED** [review]
Why is the `Other Details` section no longer showing in the transaction details screen?

**Comment by jason-gerard** [comment]
> Why is the `Other Details` section no longer showing in the transaction details screen?

Account probably isn't pro mode

**Comment by domsec** [comment]
> > Why is the `Other Details` section no longer showing in the transaction details screen?
> 
> Account probably isn't pro mode

I just realized that is probably the case lol.

**Pull request merged**